starters/basic-starter/pages/index.tsx

```diff
@@ -0,0 +1,136 @@
+}
+
+export default function IndexPage({ menus, events, places }: IndexPageProps) {
+  return (
+    <Layout title="Home" menus={menus}>
+      <div className="mt-12 lg:mt-32">
+        <section className="container mx-auto px-6">
+          <div className="w-full lg:flex items-center">
+            <div className="w-full lg:w-1/2">
+              <h2 className="text-md lg:text-2xl text-gray-600">
+                Powered by Acquia CMS
+              </h2>
+              <h1 className="text-5xl lg:text-6xl font-bold text-sky-500 mb-2 lg:mb-6">
+                A headless Next.js site
+              </h1>
+              <p className="text-md lg:text-xl font-light text-gray-800 mb-8">
+                This is placeholder text. If you are reading this, it is here by
+                mistake and we would appreciate it if you could email us with a
+                link to the page you found it on. This is placeholder text.
+              </p>
+            </div>
+            <div className="w-full lg:w-1/2 lg:pl-24">
+              <Image
+                src="/CMS_icon_0.png"
+                alt="Logo"
+                width={200}
+                height={227}
+              />
+            </div>
+          </div>
+        </section>
+      </div>
+      <div className="container px-6 pb-10 mx-auto mt-12">
+        <h2 className="text-md mb-2 lg:text-2xl text-gray-600">
+          Featured Events
+        </h2>
+        {events?.length ? (
+          <div className="grid gap-14">
+            {events.slice(0, 3).map((event) => (
+              <NodeEventTeaser key={event.id} node={event} />
+            ))}
+          </div>
+        ) : (
+          <p>No content found.</p>
+        )}
+      </div>
+      <div className="container px-6 pb-10 mx-auto mt-12 text-center items-center">
+        <h2 className="text-md mb-2 lg:text-2xl text-gray-600">Contact Us</h2>
+        {places?.length ? (
+          <div className="grid gap-14">
+            {places.slice(0, 3).map((place) => (
+              <article text-center key={place.id}>
+                <Link href={place.path.alias} passHref>
+                  <a className="no-underline hover:text-blue-600">
+                    <h2 className="text-3xl font-bold">{place.title}</h2>
+                  </a>
+                </Link>
+                <p className="text-lg text-gray-600">
+                  {place.field_place_telephone}
+                </p>
+                <p className="text-lg text-gray-600">
+                  {place.field_place_address.address_line1}
+                </p>
+              </article>
+            ))}
+          </div>
+        ) : (
+          <p>No content found.</p>
+        )}
+      </div>
+    </Layout>
+  );
+}
+
+export async function getStaticProps(
+  context,
+): Promise<GetStaticPropsResult<IndexPageProps>> {
+  const events = await getResourceCollectionFromContext<DrupalNode[]>(
+    'node--event',
+    context,
+    {
+      params: new DrupalJsonApiParams()
```

Comment:
next-drupal 1.3 introduced DrupalClient - should we be using that instead of DrupalJsonApiParams as preference?

Reply:
I think we should convert from `getResourceCollectionFromContext` to `DrupalClient`. It also utilizes `DrupalJsonApiParams` which should make it not too difficult to migrate.